### ISSUE TITLE
custom page tempaltes and custom taxonomy archive pages issue

### ISSUE THREAD

**Issue by umerkakar** [understrap/understrap#343]
Hello,
Not understand why but it's happening. Unable to figure it out. 
I have created custom post type and custom taxonomy for that post type. There I am getting an issue. 
These variables are not fetching anything from settings in those custom templates. Don't know why it's happening. On all other pages and posts templates these are working supper.

**Variables:**
$container   = get_theme_mod( 'quoteswp_container_type' );
$sidebar_pos = get_theme_mod( 'quoteswp_sidebar_position' );

Any idea why that is happening?

**Comment by umerkakar** [comment]
I just change the theme name and did as it was before. And then everything working fine. I changed the theme understrap to example after settings. But when I add new things that was creating issues. And I switched back to first theme name. And now things are working fine.

Is there a way to remove all the default settings from database. Because if we need to change the theme name after lot of other settings. Then the first theme settings saved and may be then issues comes. If we can create everything to zero without default settings. And then we change the theme name. And then new settings register under new theme name. 
Hope you understand, what I mean to say?
